docs/display_panels.rst

```diff
@@ -2,147 +2,103 @@
+**************
+The center of the window is called the workspace and it is where data is visualized. The workspace consists of display panels which display a piece of data (also called a data item). 
 
-Display Panels
---------------
-Display panels are used to show display items.
-
-You can drag items from the data panel into a display panel.
-
-You can put keyboard focus on a display panel by clicking on it.
-
-The display item with keyboard focus also serves as the primary selected item. Primary selections are indicated by a solid focus ring.
+.. image:: graphics/workspace_layout.png
+    :width: 937
+    :alt: Workspace Layout
 
-You can add secondary selected items by :kbd:`control` clicking (Windows/Linux) or :kbd:`command` (macOS) clicking on display panels. Secondary selections are indicated by a dotted focus ring.
+In a new project, the workspace will be filled by a single display panel. Use the toolbar to split the workspace into multiple smaller display panels, zoom into images, add graphics, and move an image.
 
-Processing and other commands apply to the primary selected items. Processing commands that need multiple inputs may use secondary selected items.
+To display a data item, drag it from the data panel into a display panel. You can also drag data files into a display panel directly from your desktop and from Explorer on Windows or Finder on macOS. Once the data item is in the display panel, you can use the utility panels surrounding the workspace to perform various functions, processes, manipulations, and calibrations to the selected image or line plot.
 
-To select multiple primary items, you need to use the data panel or a browser panel.
+.. image:: graphics/data_to_display.png
+    :width: 562
+    :alt: Dragging Data Items into Display Panels to Display Them
 
-.. _Display Panel Browsers:
-
-Browsers
---------
-Display panels can also show a browser, either as a thumbnail strip at the bottom of the display panel (thumbnail browser) or as a grid of display items (grid browser).
-
-You can change the browser in a display panel by using the menu items :menuselection:`Display --> Thumbnail Browser` and :menuselection:`Display --> Grid Browser`. You can change to the single display item view by choosing the menu item :menuselection:`Display --> Display Item`.
-
-You can also cycle between the display item, thumbnail, and grid browser by clicking on the display panel and pressing the :kbd:`v` key.
-
-Finally, you can change to a single display item by double clicking on it in either the thumbnail or grid browser.
-
-Note: If the display panel has an associated control bar, which is often present for live acquisition data, the :kbd:`v` key will not work.
+Different types of data are displayed in different ways such as 2D images or 1D line plots. Higher-dimensional data can be displayed as sets or lists of 2D images or as lists of 1D spectra. You can also use a display panel as a browser to help compare or manage your data.
```

Comment:
This sentence is awkward. "Different type... in different ways..." etc. Can you reword to read more easily?